Sources/FluentMongoDriver/ChangeStream.swift

```diff
@@ -0,0 +1,12 @@
+import MongoKitten
+import FluentKit
+
+extension Model {
+    public static func watch(on database: Database, options: ChangeStreamOptions = .init()) -> EventLoopFuture<ChangeStream<Self>> {
```

Comment:
Would it be possible to do something like this?
```suggestion
    public static func watch<D>(on database: D, options: ChangeStreamOptions = .init()) -> EventLoopFuture<ChangeStream<Self>> where D: Database & MongoDatabaseRepresentable {
```

No clue if that works & compiles, but might allow you to remove the runtime check into a compile time one :)

Reply:
Yes, I did that at first actually. I thought it might make the API too burdensome

Reply:
Actuall, come think of it. This wouldn't do anything, because 9/10 times you just use `app.db` or `req.db` anyways, which will be type-erased to just `Database`. Nevermind :)

Reply:
Exactly. I think it would be correct but burdensome :)